projects/kit/components/input-range/input-range.style.less

```diff
@@ -1,3 +1,5 @@
+@track-thickness: calc(var(--tui-thumb-radius) / 3);
```

Comment:
Should we create constant inside `projects/core/styles/mixins/slider.{less,scss}` and share it between `Slider`, `Range` and `InputRange` ?